Firmware/FFBoard/Src/SPIButtons.cpp

```diff
@@ -5,50 +5,56 @@
  *      Author: Yannick
  */
 
+#include <math.h>
+#include <tuple>
+
 #include "constants.h"
 #include <SPIButtons.h>
 #include "eeprom_addresses.h"
-#include "math.h"
 
-ClassIdentifier SPI_Buttons::info = {
-		 .name = "SPI" ,
+static ButtonSourceConfig decodeIntToConf(uint16_t config, uint16_t config_int_2);
+static std::tuple<uint16_t, uint16_t> encodeConfToInt(ButtonSourceConfig* c);
+
+ClassIdentifier SPI_Buttons_1::info = {
+		 .name = "SPI 1" ,
```

Comment:
Would probably change the names to something more meaningful to not cause confusion with the SPI ports.
"SPI Btn 1" or "SPI Buttons 1" maybe to make it clear that the selections are just 2 independent instances instead of different SPI ports.

Reply:
👍